dolphinscheduler-datasource-plugin/dolphinscheduler-datasource-kyuubi/src/main/java/org/apache/dolphinscheduler/plugin/datasource/kyuubi/KyuubiDataSourceClient.java

```diff
@@ -59,7 +67,7 @@ public Connection getConnection() {
         Connection connection = null;
         while (connection == null) {
             try {
-                connection = dataSource.getConnection();
+                connection = driverManagerDataSource.getConnection();
```

Comment:
Please add max retry times here, this is dangerous code.

Reply:
Got it

Reply:
done.

Reply:
Could plz take a look at this ? Thanks.